packages/docs-page/server/loaders/remote-content.test.ts

```diff
@@ -256,92 +258,48 @@ describe('RemoteContentLoader', () => {
 
 describe('mapVersionList', () => {
   const versionMetadataList = versionMetadata_200.result
-  const versionList = mapVersionList(versionMetadataList)
+  const versionList = mapVersionList(
+    versionMetadataList as VersionMetadataItem[]
+  )
 
-  test('should label the first item as "latest"', () => {
-    expect(versionList[0].name).toEqual('latest')
-    expect(versionList[0].label.endsWith('(latest)')).toBe(true)
+  test('should label the latest version as "latest"', () => {
+    const latestVersion = versionList.find(
+      (version) => version.isLatest === true
+    )
+
+    expect(latestVersion?.label.endsWith('(latest)')).toBe(true)
   })
 
-  test('should sort by semver descending', () => {
```

Comment:
👍 (This sorting logic predates sorting at the API level. This is good to remove)